ui/hooks/accounts/useMultichainWalletSnapClient.ts

```diff
@@ -20,6 +20,10 @@ import {
   SOLANA_WALLET_NAME,
   SOLANA_WALLET_SNAP_ID,
 } from '../../../shared/lib/accounts/solana-wallet-snap';
+import {
+  TRON_WALLET_NAME,
+  TRON_WALLET_SNAP_ID,
+} from '../../../shared/lib/accounts/tron-wallet-snap';
```

Comment:
no codefences

Reply:
I didn't see one for Bitcoin so I didn't add it. But I can definitely add it, you think it's a good idea?

Reply:
lets add it I think